packages/components/tabs/src/TabsRoot.tsx

```diff
@@ -0,0 +1,65 @@
+import * as RadixTabs from '@radix-ui/react-tabs'
+import { forwardRef, type PropsWithChildren, useState } from 'react'
+
+import { TabsContext, type TabsContextInterface } from './TabsContext'
+import { rootStyles } from './TabsRoot.styles'
+import type { TabsTriggerVariantsProps } from './TabsTrigger.styles'
+
+export interface TabsRootProps
+  extends Omit<RadixTabs.TabsProps, 'activationMode'>,
+    PropsWithChildren<TabsTriggerVariantsProps> {
+  /**
+   * Change the component to the HTML tag or custom component of the only child. This will merge the original component props with the props of the supplied element/component and change the underlying DOM node.
+   * @default false
+   */
+  asChild?: boolean
+}
+
+export const TabsRoot = forwardRef<HTMLDivElement, TabsRootProps>(
+  (
+    {
+      intent = 'primary',
+      size = 'md',
+      /**
+       * Default Radix Primitive values
+       * see https://www.radix-ui.com/docs/primitives/components/tabs#root
+       */
+      asChild = false,
+      orientation = 'horizontal',
+      children,
+      className,
+      defaultValue,
+      value,
+      onValueChange,
+    },
+    ref
+  ) => {
+    const [selectedTab, setSelectedTab] = useState<TabsContextInterface['selectedTab']>(
+      value || defaultValue
+    )
+
+    const handleChange = (v: string) => {
+      onValueChange?.(v)
+      setSelectedTab(v)
+    }
```

Comment:
```suggestion
    const handleChange = (value: string) => {
      onValueChange?.(value)
      setSelectedTab(value)
    }
```

Pressing keys has no cost. Name variables with a deterministic relevant name.

Reply:
`e` can be error or event or ...
`d` can be data or digit or decimal or ...
`v` can ve value or version or ...